data/preset/escola.yaml

```diff
@@ -1,19 +1,33 @@
 ---
-defines:
+vars:
+  app-banner:
+    shortdef: 'Banner.Dynamic(mode: text, expand: true)'
+  app-brand:
+    shortdef: 'ContentGroup.StaticText(label: "TV ESCOLA")'
```

Comment:
Oh, I had forgotten about this. Perfect use case for variables! It would be even better to make this empty by default, and put "TV ESCOLA" in the app.yaml files.

Reply:
Sure, I will empty the string.

Reply:
Totally agree! But, I recommend that maybe we make a separate ticket to do a larger audit of what the "default template" settings should be; I feel like there will be more things that we'd like to change as default.

But I guess this is such a small & obvious one, that changing it now wouldn't hurt.